lstchain/scripts/lstchain_dl1_to_dl2.py

```diff
@@ -63,6 +63,11 @@
                     help='Path to a configuration file. If none is given, a standard configuration is applied',
                     default=None, required=False)
 
+parser.add_argument('--tel-id', '-tid', action='store', type=str,
```

Comment:
Why this parameter is needed? The telescopes to be read out can be retrieved from the JSON config file which is already an argument here.

Reply:
from which entry in the config?

Reply:
Though the indexing problem is actually deeper. See the conversation here: https://github.com/cta-observatory/cta-lstchain/issues/633